website/docs/docs/cloud/secure/environment-permissions-setup.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Set up environment-level permissions"
+id: environment-permissions-setup
+description: "Set up environment-level permissions to protect your information"
+sidebar_label: "Set up environment-level permissions"
+pagination_next: null
+pagination_prev: null
+---
+
+# Set up environment-level permissions <Lifecycle status='beta' />
+
+:::note
+
+This is a beta feature available to select dbt Cloud Enterprise customers. If you are interested in beta testing this feature, please contact your account manager.
+
+:::
+
+To set up and configure environment-level permissions, you must have write permissions to the **Groups & Licenses** settings of your dbt Cloud account. For more information about roles and permissions, check out [User permissions and licenses](/docs/cloud/manage-access/seats-and-users).
+
+Environment-level permissions are not the same as account-level [role-based access control (RBAC)](/docs/cloud/manage-access/about-user-access#role-based-access-control) and are configured separately from those workflows.
+
+## Setup instructions
+
+In your dbt Cloud account:
+
+1. Open the **gear menu** and select **Account settings**. From the left-side menu, select **Groups & Licenses**. While you can edit existing groups, we recommend not altering the default `Everyone`, `Member`, and `Owner` groups.
+
+<Lightbox src="/img/docs/dbt-cloud/groups-and-licenses.png" width="80%" title="Groups & Licenses page in dbt Cloud with the default groups highlighted."/>
+
+2. Create a new or open an existing group. If it's a new group, give it a name, then scroll down to **Access & permissions**. Click **Add**.
+
+<Lightbox src="/img/docs/dbt-cloud/add-permissions.png" width="80%" title="The Access & permissions section with the Add button highlighted."/>
+
+3. Select the **Permission set** for the group. Only the following permissions sets can have environment-level permissions configured:
+
+- Database admin
+- Git admin
+- Team admin
+- Analyst
+- Developer
+
+Other permission sets are restricted because they have access to everything (for example, Account admin), or limitations prevent them from having write access to environments (for example, Account viewer).
+
+If you select a permission set that is not supported, the environment permission option will not appear.
+
+<Lightbox src="/img/docs/dbt-cloud/no-option.png" width="80%" title="The view of the permissions box if there is no option for environment permissions."/>
+
+4. Select the **Environment** for group access. The default is **All environments**, but you can select multiple. If none are selected, the group will have read-only access. Note that `Other` maps to the `General` environment type. 
```

Comment:
We're planning to update "Other" to say "General" in the dropdown so it's less confusing. Since that hasn't been updated yet, we can keep this note for now but will want to remove it when we make that update